### ISSUE TITLE
recording: burial module with bone recording

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/arch-kiosk-office#1982]
The idea would be to have an interactive skeleton drawing (or several because one needs different angles?) in which single bones can be selected.

- Apart from just found or not found, what about additional data per bone?
- to turn this into actual data we need a name (a unique identifier) for each bone (Oh je, how many single bones does a human skeleton have?). I suspect medicine has already standardized such names?
- what about other skeletons? Do you find and reconstruct cats?
- what about finding children or adolescents? Would it make sense to have different skeletons? 
- Research: Are there already skeleton models (ontologies) out there that we could use to reduce our work?

**Comment by lbestock** [comment]
We won't need different angles but there will be at least two, one for adults and one for people whose epiphyses haven't fused yet. People to talk to: Rachel Kalisher (her Excel files for OCHRE we already have and those have bone names I believe - she does in table form what some do visually). Brenda Baker.

**Comment by lbestock** [comment]
To do this properly, it's not just like the soil sorting. As this form indicates and as is standard on skeletal inventory recording, it isn't just presence/absence of a bone but how much of a bone is present. Like this:
![image](https://user-images.githubusercontent.com/49291909/219969603-bece09dd-b8ce-48a5-882a-127b1b19ebc7.png)
That isn't clicking on bones, it's coloring in bones, some partially.

**Comment by urapadmin** [comment]
ja. Nice. But that would take me weeks and it indeed does not really translate into distinct data. It isn't only about the percentage of the bone it is even showing the part of the bone that is preserved. That's too sophisticated, I am afraid. And it would need a fully fledged drawing tool.

**Comment by urapadmin** [comment]
seems we need a full-time CV developer.

**Comment by urapadmin** [comment]
hm. I have to think about it. Perhaps it is possible. It would be cool, wouldn't it?

**Comment by urapadmin** [comment]
@lbestock: That's another perfect example of a module that could be given to a (gifted) student as an isolated project. It is easy to describe and does not need Kiosk background knowledge.

**Comment by lbestock** [comment]
I have reached out to Brenda Baker to talk about this. I told her I wanted to address two things: her own burial recording and its suitability for iPadding, and what would constitute best practice for a team who had no physical anthropologist that season but hit burials and needed to record them as well as possible so that a future physical anthropological study wouldn't be hamstrung. I know very well that the second part will start with adamant insistence that projects should immediately shut down rather than even look at a burial if there is no physical anthropologist present, and that Brenda and I will still be talking once we get past that point. It's helpful to have MKAP to think about as a real scenario - not intending to dig burials, might hit them. Until we have a fully formed tool here, could we have a locus details page that spurs excavators to collect some data in a way that will be useful and that will not mean reduplicating our own efforts down the road?

**Comment by urapadmin** [comment]
what would it look like?

**Comment by lbestock** [comment]
I have a zoom with Brenda next week. She has sent her own burial forms, which she still does on paper and which are now in the googledrive. BUT she has also given me a contact for a colleague who uses FileMaker for burial recording and I will get in touch with her immediately.

**Comment by lbestock** [comment]
Very good initial meeting with Brenda who is excited about this, though still incredulous that we used iPads at Uronarti without massive equipment failure. She is going to send me some filled in forms of hers, and she is more than happy if we want to use her forms as the basis of a basic burial recording module. (The other person hasn't gotten back to me yet.) She herself changes forms based on what site she is working at (in Egypt she has fields about coffins; in Sudan she does not, for example) so she was pleased to hear that our whole approach is accommodating workflow and specialist differences rather than imposing a norm. But she still thinks having a basic form is important, particularly as sometimes (MKAP) there is no intent to dig a burial but one is hit, and without a bioarchaeologist there are some types of data that the excavators just don't know to collect, like elevations that are specific to burial recording.

I have to run, but a couple of interesting things came out of discussion about the image of the bones, which she uses extensively. First, coloring a bone in can be EITHER presence or absence. If it's well preserved, she only marks the absent ones. If it's poorly preserved, she marks the present ones. Easy for us to imagine accommodating, turn them all black or all white to start, then click the ones you want to exclude. I don't know about if she ever colors half a bone, a bigger problem. But also she does something else important with the drawing - she makes double slashes at points of disarticulation. For instance, most of the bodies she digs in Sudan have disarticulated heads because someone has gotten in there to loot necklaces. She got quite excited about the possibility of having that as data, of being able to ask immediately what percent of bodies have disarticulated heads, if there are patterns by sex or age. I can imagine it with a head (disarticulation button on the drawing that draws two lines, making it both visual and a yes/no data point?) But if she drops those things all over skeletons then that gets way trickier.

**Comment by lbestock** [comment]
Ha ha. I now have a bunch of filled in forms from Brenda. And thinking of the skeleton image as leading to data is really a non-starter:
![image](https://github.com/arch-kiosk/arch-kiosk-office/assets/49291909/03a9be9c-a653-40af-8ff0-ad116ef5dfe5)

**Comment by urapadmin** [comment]
one should write a paper "on data" that clarifies what data actually is

**Comment by urapadmin** [comment]
or rather what is not

**Comment by urapadmin** [comment]
The Archaeology that I see is at least not working with discrete data. 
But let's not make the mistake and think Brenda could not do it otherwise. I mean, as long as you use paper, there is not reason to use it in a more discrete way. We have to find out a) where the paper isn't good at and b) what kind of results (analysis) the paper cannot easily yield but would be really cool. Then they might be willing to use a software that does not allow for the same rogue annotation (or at least not if one wants a tool that can do the cool thing that paper can't). Paper is great on many a front. Copying it just to make it digital is not worth it.

**Comment by lbestock** [comment]
At least I think Brenda is someone with whom it could be discussed. She was immediately open to my idea of clicking a button to place neck disarticulation, not showing it visually as she was used to. I thanked her for sending me all the filled in forms and noted that one thing digital recording does is make it so you never have to read anyone's handwriting again and she laughed and replied that she chose which forms to send me based on which were most legible, because some are pure chicken scratch. The only analysis point we hit so far was on disarticulated heads and patterns thereof, but she does try to answer population level questions and so maybe that discussion could get farther with her than with the diggers.

**Comment by urapadmin** [comment]
this is not going anywhere conceptually but also not on my front: There will be no image annotation module in 2024. I have to lay the groundwork first. Just to let everybody know who might be counting on it.

**Comment by urapadmin** [comment]
- see #2690
